verl/trainer/ppo/ray_trainer.py

```diff
@@ -467,26 +495,22 @@ def _create_dataloader(self):
 
         self.train_dataset = dataset_cls(
             data_files=self.config.data.train_files,
+            data_file_weights=self.config.data.get("train_file_weights", None),
```

Comment:
no need to add additional arguments since config itself is passed to dataset

Reply:
I see. But that way, the `dataset_cls` has to tell whether it is called for training (can be a list of files, see [code here](https://github.com/volcengine/verl/blob/ea4cd319871425df7d8a9b600a177cce185fe3ec/verl/trainer/ppo/ray_trainer.py#L430)) or for validating (can be only a single one dataset, [code here](https://github.com/volcengine/verl/blob/ea4cd319871425df7d8a9b600a177cce185fe3ec/verl/trainer/ppo/ray_trainer.py#L454)), which requires an extra config arg called, say, `mode = 'train' or 'valid' `... So I guess it could be more elegant to pass the `dataset_file_weights` directly?